perllib/FixMyStreet/App/Form/Waste/Garden.pm

```diff
@@ -157,6 +180,14 @@ has_field bins_wanted => (
     },
 );
 
+has_field apply_discount => (
+    type => 'Checkbox',
+    label => 'Customer discount',
+    option_label => FixMyStreet::Template::SafeString->new(
```

Comment:
No harm in being this, just to note that the reason some other option_label use SafeString is because they have HTML that shouldn't be escaped in them, which isn't the case here. Should it perhaps give the % figure discount?